Description: ada/terminals.c: remove obsolete termio.h
 On all architectures, the terminals.c source file #includes
 <termios.h> and declares variables with type struct termios.
 .
 Some platforms provide a compatibility termio.h, which only defines
 the termio structure.
 .
 terminals.c also #includes <termio.h>, probably for historical
 reasons since no termio structure is ever used.
 .
 Drop the #include instead of maintaining a list of architectures.
Author: Nicolas Boulenguez <nicolas@debian.org>
Bug-Debian: https://bugs.debian.org/845159
Forwarded: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=81103

Index: b/src/gcc/ada/terminals.c
===================================================================
--- a/src/gcc/ada/terminals.c
+++ b/src/gcc/ada/terminals.c
@@ -1107,14 +1107,6 @@ __gnat_setup_winsize (void *desc, int ro
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
-
-/* On some system termio is either absent or including it will disable termios
-   (HP-UX) */
-#if !defined (__hpux__) && !defined (BSD) && !defined (__APPLE__) \
-  && !defined (__rtems__) && !defined (__QNXNTO__)
-#   include <termio.h>
-#endif
-
 #include <sys/ioctl.h>
 #include <termios.h>
 #include <fcntl.h>
@@ -1130,7 +1122,6 @@ __gnat_setup_winsize (void *desc, int ro
 #   include <sys/signal.h>
 #endif
 #if defined (__hpux__)
-#   include <sys/termio.h>
 #   include <sys/stropts.h>
 #endif
 
